x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/slm/SnapshotLifecycleInitialisationTests.java

```diff
@@ -44,7 +45,7 @@ public static void setupRepositoryPath() {
     @Override
     protected Settings nodeSettings() {
         Settings.Builder settings = Settings.builder().put(super.nodeSettings());
-        settings.put(XPackSettings.INDEX_LIFECYCLE_ENABLED.getKey(), false);
+        settings.put(LifecycleSettings.LIFECYCLE_HISTORY_INDEX_ENABLED_SETTING.getKey(), false);
```

Comment:
Can you explain this change, just so I can better understand?

Reply:
The idea is that the lifecycle history index is irrelevant to these tests, so it's less setup and teardown.

Reply:
The tests run fine with this setting enabled, so I'll drop this line.